.github/workflows/linux_unit_tests_with_latest_deps.yaml

```diff
@@ -15,6 +15,8 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - python_version: "3.9"
+            command: 'git-test-automl'
```

Comment:
Do we need to test on 3.9? If so, should we just bump up the Python versions to 3.9 across all tests?

Reply:
Testing on 3.9 is how we can double check that the pinning worked, since pandas 2.1.0 isn't installed on 3.8. I don't think we should just immediately bump everything up, but I do agree that arbitrarily just running `git-test-automl` for 3.9 and not the others doesn't make much sense.

Thoughts on testing 3.9 alongside 3.8 for all PR tests? I don't think it'll add any time, since the tests all run in parallel

Reply:
I think adding one additional test for 3.9 is sufficient.